.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.6.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Users/azdrojowa/Library/Java/JavaVirtualMachines/corretto-17.0.3/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/azdrojowa/Library/Application Support/JetBrains/IdeaIC2022.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/azdrojowa/Library/Application%20Support/JetBrains/IdeaIC2022.1/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
Pls add `.bsp/` to .gitignore (actually... weird that it wasn't ignore already 🤔)

Reply:
yes did it now